docs/manual/releasing.rst

```diff
@@ -0,0 +1,34 @@
+===============
+Releasing Qtile
+===============
+
+Here is a short document about how to tag Qtile releases.
+
+1. create a "Release vX.XX.XX" commit. I like to thank all the contributors; I
+   find that list with something like:
+
+.. bash::
+
+   git log --oneline --no-merges --format=%an $(git describe --tags --abbrev=0)..HEAD | sort -u
+
+note that this can sometimes generate duplicates if people commit with slightly
+different names; I typically clean it up manually.
+
+Be sure that you GPG-sign (i.e. the ``-S`` argument to ``git commit``) this commit.
+
+2. Create a GPG-signed annotated tag (``git commit -a -s``); I usually just use
+   exactly the same commit message as the actual release commit above.
+
+3. Push your release commit to git master and your tag.
```

Comment:
Do you need to say you're pushing straight to upstream rather than your fork? Would be good to give command line here too.